app/renderer/src/components/ResultItem.js

```diff
@@ -67,14 +67,19 @@ const ResultItem = ({ theme, selected, item, onDoubleClick }) => {
     )
     : {};
 
+  // apply modifiers if necessary
+  const itemSubtitle = (isSuperMod && item.mods && item.mods.cmd && item.mods.cmd.subtitle)
+    || (isAltMod && item.mods && item.mods.alt && item.mods.alt.subtitle)
+    || item.subtitle;
+
   return (
     <li {...compose(base, themeBase, themeHover, themeSelected)} onDoubleClick={onDoubleClick}>
       <div {...icon}>
         <Icon icon={item.icon} />
       </div>
       <div {...details}>
         <h2 {...compose(title, theme.resultTitle || {})}>{item.title}</h2>
-        <h3 {...compose(subtitle, theme.resultSubtitle || {})}>{item.subtitle}</h3>
+        <h3 {...compose(subtitle, theme.resultSubtitle || {})}>{itemSubtitle}</h3>
```

Comment:
why the fallback `||`?
I thought that `compose` by default gives a `{}` incase of null/falsy values 😜